app/components/assets/form.tsx

```diff
@@ -371,6 +374,28 @@ export const AssetForm = ({
           </div>
         </FormRow>
 
+        <FormRow
+          rowLabel={"Property ID"}
+          subHeading={
+            <p>
+              Specify the property ID of your asset.
```

Comment:
I think we should add some more info about this. Like what is the purpose of this field. I think some of our users might be confused especially if they are not familiar with the terminology.